app/api/v1/antenna/route.ts

```diff
@@ -3,7 +3,10 @@ import { pool } from '../connection';
 
 export async function GET() {
   try {
+    console.log('Connecting...');
```

Comment:
@JonMike8 What is the purpose of the console.logs?

Reply:
i think i put them for debugging purposes earlier. i can take them out if you want

Reply:
Yes please, if they are only for debugging it'd be better to take them off. Later on we can implement middleware that will log request level info.

Reply:
@JonMike8 You also have linter errors. To fix them, you can run:
```
npm run lint
```
and

```
npm run format:fix
```